internal/content_coding.go

```diff
@@ -3,10 +3,11 @@ package internal
 import (
 	"bufio"
 	"bytes"
-	"compress/gzip"
-	"compress/zlib"
 	"errors"
 	"fmt"
+	"github.com/klauspost/compress/zlib"
+	"github.com/klauspost/compress/zstd"
+	gzip "github.com/klauspost/pgzip"
 	"io"
```

Comment:
We group import by _built-in_, _3rd party_ and _telegraf-internal_, each group sorted alphabetically and separated by a newline. So you should do
```suggestion
	"io"

	"github.com/klauspost/compress/zlib"
	"github.com/klauspost/compress/zstd"
	gzip "github.com/klauspost/pgzip"
```